build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -106,7 +106,7 @@ public override string ToString()
         // These versions are used when generating platform installers.
         //
         public bool IsLocked = true; // Set this variable to toggle muxer locking.
-        public VerInfo LockedHostFxrVersion => IsLocked ? new VerInfo(1, 0, 1, "", "", "", CommitCountString) : LatestHostFxrVersion;
+        public VerInfo LockedHostFxrVersion => IsLocked ? new VerInfo(1, 0, 2, "", "", "", CommitCountString) : LatestHostFxrVersion;
```

Comment:
This issue is really a platform installer bug and should not require a nuget package update (doing so should happen only if we fixed a bug in hostfxr and want to fix a new one).

Can we force a version just for the platform installer?